ps8/phpstan.neon

```
@@ -18,4 +19,7 @@ parameters:
 	    - ../presentation/vendor
 	    - ../utility/vendor
 	scanDirectories:
-		- vendor/stancer/php-stubs-prestashop/stubs/
\ No newline at end of file
+		- vendor/stancer/php-stubs-prestashop/stubs/
+	ignoreErrors:
+	    -
+	        identifier: offsetAccess.nonOffsetAccessible
```

I am against it too, ignoring this identifier makes PHPStan almost irrelevant when heavily using arrays, which is our case.

---

It's also irrelevant when we ignore all errors related to it. Removed.